File: app/src/main/java/com/ty/zbpet/ui/base/BaseResponse.java
package com.ty.zbpet.ui.base;

import java.io.Serializable;

/**
 * @author TY on 2018/10/26.
 */
public class BaseResponse<T> implements Serializable {

    private String tag;
    private String message;
    private T data;

    public String getTag() {
        return tag;
    }

    public void setTag(String tag) {
        this.tag = tag;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public T getData() {
        return data;
    }

    public void setData(T data) {
        this.data = data;
    }
}

File: app/src/main/java/com/ty/zbpet/ui/base/BaseResponse.kt
package com.ty.zbpet.ui.base

import java.io.Serializable

/**
 * @author TY on 2018/10/26.
 * 响应基类
 */
class BaseResponse<T> : Serializable {

    var tag: String? = null
    var message: String? = null
    var data: T? = null
}
